test/integration-test-setup.sh

```diff
@@ -24,7 +24,7 @@ case "$1" in
                     pacman --upgrade --needed --noconfirm /work/build/*.pkg.tar
                     ;;
                 debian|ubuntu)
-                    apt-get install /work/build/*.deb
+                    DEBIAN_FRONTEND=noninteractive apt -y install /work/build/*.deb
```

Comment:
Hmm, the commit is silent on this, but this changes `apt-get` to `apt`. Please explain.

Reply:
@keszybz interesting, do you remember why?

Reply:
`apt-get` has [stronger compatibility guarantees](https://manpages.debian.org/bookworm/apt/apt.8.en.html#SCRIPT_USAGE_AND_DIFFERENCES_FROM_OTHER_APT_TOOLS) whereas `apt` is mostly intended for interactive use and may change some behaviors and defaults if they deem them sensible.

Reply:
this is not really the case in reality, `apt install -y` will never break